prep-javascript-dom-manipulation/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript DOM Manipulation</title>
+  <style>
+    #container {
+      border: 2px solid green;
+    }
+  </style>
+</head>
+<body>
+  <div id="container">
+    <p class="first-paragraph">
+      I really like coding.
+    </p>
+    <p class="second-paragraph">
+      I also really like to watch television and eating pancakes.
+    </p>
+    <p class="third-paragraph">
+      I hope one day to meet people that also like coding, television, and pancakes.
+    </p>
+  </div>
```

Comment:
Hi Nick, try removing the text content from your html for the second and third paragraphs. If the JavaScript is working, it will add it for you when the page loads.